packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -64,31 +88,61 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
 
       if (normalTexture != null && normalMap != null) {
         normalTextures.add(normalMap);
+      } else {
+        normalTexture = new EmptyNormalTexture();
```

Comment:
Can you just write `normalTexture = {index: -1};`? And remove all the `EmptyTexture`s?

Reply:
Oh, looks like you can, learning what typescript lets you get away with is a WIP